### ISSUE TITLE
Fix hidden breadcrumb with no focus indicator can be accessed via keyboard navigation

### ISSUE THREAD

**Issue by e-marie-w** [uiowa/uiowa#9417]
# Background

When keyboard navigating through a page like https://stories.uiowa.edu/writing-across-disciplines-among-elite-company-usnwr-rankings-2025-26 or https://now.uiowa.edu/resources, the keyboard focus indicator disappears upon tabbing past the search bar. It reappears if you tab again. 

Based on EoD discussion, this is because of the hidden breadcrumb block. On the homepage https://stories.uiowa.edu/, this behavior does not happen because the breadcrumb block is not rendered on the homepage. 

# Story

As a SiteNow visitor who navigates sites via keyboard, I should always see a visible focus indicator of where I am on a page. 

# Proposed Solution
Do not render breadcrumb block when it is hidden?
